client/extensions/woocommerce/app/orders/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'state/ui/selectors';
```

Comment:
Would the fallback selector be more robust here?  https://github.com/Automattic/wp-calypso/blob/e1f9cddd82b2ca92e1e3f20f7cbec36812ea088b/client/extensions/woocommerce/state/sites/selectors.js#L16